docs/spec/spec.md

```diff
@@ -314,69 +280,55 @@ metadata:
   selfLink: ...
   ...
 
-# spec contains one of several possible rollout styles
-spec:  # One of "runLatest", "release", "pinned" (DEPRECATED), or "manual"
-
-  # Example, only one of "runLatest", "release", "pinned" (DEPRECATED), or "manual" can be set in practice.
-  runLatest:
-    configuration:  # serving.knative.dev/v1alpha1.ConfigurationSpec
-      # +optional. The build resource to instantiate to produce the container.
-      build: ...
-      revisionTemplate: ... # Deprecated way of expressing the same as template.
-      template:
-        metadata:
-          # Name may be specified here following the same rules as Configuration.
-          # When the name collides, the Route must not be updated to the
-          # conflicting Revision.
-        spec: # serving.knative.dev/v1alpha1.RevisionSpec
-          container: ... # See the Container section below
-          volumes: ... # Optional
-          containerConcurrency: ... # Optional
-          timeoutSeconds: ... # Optional
-          serviceAccountName: ...  # Name of the service account the code should run as
-
-  # Example, only one of "runLatest", "release", "pinned" (DEPRECATED), or "manual" can be set in practice.
-  pinned:
-    revisionName: myservice-00013  # Auto-generated revision name
-    configuration:  # serving.knative.dev/v1alpha1.ConfigurationSpec
-      # +optional. The build resource to instantiate to produce the container.
-      build: ...
-      revisionTemplate: ... # Deprecated way of expressing the same as template.
-      template:
-        spec: # serving.knative.dev/v1alpha1.RevisionSpec
-          container: ... # See the Container section below
-          volumes: ... # Optional
-          containerConcurrency: ... # Optional
-          timeoutSeconds: ... # Optional
-          serviceAccountName: ...  # Name of the service account the code should run as
-
-  # Example, only one of "runLatest", "release", "pinned" (DEPRECATED), or "manual" can be set in practice.
-  release:
+spec:
+  template:
+    metadata:
+      # Name may be specified here following the same rules as Configuration.
+      # When the name collides, the Route must not be updated to the
+      # conflicting Revision.
+    spec:
+      # Service supports the full inline Configuration spec.
+      containers: ... # See the Container section below
+      volumes: ...
```

Comment:
I am going to make an issue as I don't think it needs to be fixed here, but it would be good to be able to go a level deeper into these "..." as with volumes there are only certain things you can do which are not obvious from here or any of our other documentation.

Reply:
Ack